tests/meltano/cli/test_cli.py

```diff
@@ -65,6 +68,7 @@ def test_activate_project_readonly_env(
 
         assert Project._default.readonly
 
+    @pytest.mark.order(2)
```

Comment:
Should this be `3`?

```suggestion
    @pytest.mark.order(3)
```

or does it mean the two tests with `2` can run in any order?

Reply:
Tests with a particular number can run in any order relative to other tests with that same number. The orders are scoped per-module.